packages/metrics/src/features/metrics/utils.ts

```diff
@@ -479,3 +510,25 @@ export function getPopulation(
 
   return totalPopulation ? totalPopulation[populationYearToConsider] : 0
 }
+
+function getDaysPerYear(fromDate: Date, toDate: Date) {
+  const daysPerYear = []
+  const allDates = eachDayOfInterval({ start: fromDate, end: toDate })
+  let currentYear = getYear(fromDate)
+  let daysInYear = 0
+
+  allDates.forEach((date) => {
+    const year = getYear(date)
+    if (year !== currentYear) {
+      daysPerYear.push(daysInYear)
+      currentYear = year
+      daysInYear = 0
+    }
+
+    daysInYear++
+  })
+
+  daysPerYear.push(daysInYear)
+
+  return daysPerYear
+}
```

Comment:
We should be able to do this a bit more efficiently with https://date-fns.org/v2.30.0/docs/eachYearOfInterval